base/pkg/entry.jl

```diff
@@ -206,21 +206,20 @@ function clone(url::AbstractString, pkg::AbstractString)
     end
 end
 
-function clone(url_or_pkg::AbstractString)
-    urlpath = joinpath("METADATA",url_or_pkg,"url")
-    if !(':' in url_or_pkg) && isfile(urlpath)
-        pkg = url_or_pkg
-        url = readchomp(urlpath)
-        # TODO: Cache.prefetch(pkg,url)
```

Comment:
Then it should be added back with more explanation because I had no idea what it meant and I originally wrote this code.